.github/workflows/push.yaml

```diff
@@ -0,0 +1,121 @@
+name: Push to master
+run-name: ${{ github.ref_name }} push run 🚀
+on:
+  push:
+    branches:
+      - master
+      - 'release/**'
```

Comment:
The default branch name by GitHub (and maybe even git?) is `main` now so it makes sense to include that here (and in other places, e.g. the lines below and the pr-pull file) as well.